src/components/Result.tsx

```diff
@@ -37,11 +43,30 @@ const Result = () => {
     };
 
     return (
-        <div>
-            <p>Question : {result.question.questionNumber}</p>
-            <p>Your answer : {answerId}</p>
-            <p>Answer 1: %{result.firstAnswerRate}</p>
-            <p>Answer2: %{result.secondAnswerRate}</p>
+        <div className='result__container'>
```

Comment:
should be just  className='result' based on the names you gave the children divs.